wasp/src/main/java/com/orhanobut/wasp/VolleyImageNetworkHandler.java

```diff
@@ -0,0 +1,99 @@
+package com.orhanobut.wasp;
+
+import android.content.Context;
+import android.graphics.Bitmap;
+
+import com.android.volley.NetworkResponse;
+import com.android.volley.Request;
+import com.android.volley.RequestQueue;
+import com.android.volley.Response;
+import com.android.volley.VolleyError;
+import com.android.volley.toolbox.HttpHeaderParser;
+import com.android.volley.toolbox.ImageRequest;
+import com.android.volley.toolbox.Volley;
+import com.orhanobut.wasp.utils.StringUtils;
+
+import java.io.UnsupportedEncodingException;
+
+/**
+ * @author Orhan Obut
+ */
+public class VolleyImageNetworkHandler implements WaspImageHandler.ImageNetworkHandler {
+
+    private final RequestQueue requestQueue;
+
+    public VolleyImageNetworkHandler(Context context) {
+        this.requestQueue = Volley.newRequestQueue(context);
```

Comment:
What do you think about using httpStack given to the Wasp.Builder here also? Currently at the time of creating an object from this class we do not have access to the given httpStack but I think we can initialise it when building wasp, can't we?